frontend/public/components/dashboard/project-dashboard/inventory-card.tsx

```diff
@@ -144,46 +153,55 @@ export const InventoryCard = () => {
-        <ProjectInventoryItem projectName={projectName} model={ConfigMapModel} />
-        {canListSecrets && <ProjectInventoryItem projectName={projectName} model={SecretModel} />}
-        {itemExtensions.map((item) => (
+        <Stack hasGutter>
+          <ProjectInventoryItem projectName={projectName} model={DeploymentModel} />
+          <ProjectInventoryItem projectName={projectName} model={DeploymentConfigModel} />
+          <ProjectInventoryItem projectName={projectName} model={StatefulSetModel} />
           <ProjectInventoryItem
-            key={item.properties.model.kind}
             projectName={projectName}
-            model={item.properties.model}
-            mapper={item.properties.mapper}
-            additionalResources={item.properties.additionalResources}
+            model={PodModel}
+            mapper={getPodStatusGroups}
           />
-        ))}
-        {dynamicItemExtensions.map((item) => (
           <ProjectInventoryItem
-            key={item.properties.model.kind}
             projectName={projectName}
-            model={item.properties.model}
-            mapper={item.properties.mapper}
-            additionalDynamicResources={item.properties.additionalResources}
+            model={PersistentVolumeClaimModel}
+            mapper={getPVCStatusGroups}
           />
-        ))}
-        <ProjectInventoryItem
-          projectName={projectName}
-          model={VolumeSnapshotModel}
-          mapper={getVSStatusGroups}
-        />
+          <ProjectInventoryItem projectName={projectName} model={ServiceModel} />
+          <ProjectInventoryItem projectName={projectName} model={RouteModel} />
+          <ProjectInventoryItem projectName={projectName} model={ConfigMapModel} />
+          <ProjectInventoryItem
+            projectName={projectName}
+            model={VolumeSnapshotModel}
+            mapper={getVSStatusGroups}
+          />
+          {canListSecrets && <ProjectInventoryItem projectName={projectName} model={SecretModel} />}
+          {itemExtensions.map((item) => (
+            <ProjectInventoryItem
+              key={item.properties.model.kind}
+              projectName={projectName}
+              model={item.properties.model}
+              mapper={item.properties.mapper}
+              additionalResources={item.properties.additionalResources}
+            />
+          ))}
+          {dynamicItemExtensions.map((item) => (
+            <ProjectInventoryItem
+              key={item.properties.model.kind}
+              projectName={projectName}
+              model={item.properties.model}
+              mapper={item.properties.mapper}
+              additionalDynamicResources={item.properties.additionalResources}
+            />
+          ))}
+          {inventoryExtensions.map(({ uid, properties: { component: Component } }) => (
+            <StackItem key={uid}>
+              <ErrorBoundary>
+                <Component projectName={projectName} />
```

Comment:
If the `Component` errors out -- you'll end up with `<StackItem><div /></StackItem>` -- which may not be the desired effect as `Stack` with `hasGutter` will cause whitespace for each of these.

Recommend bringing your `ErrorBoundary` above `StackItem`.

Reply:
Thanks @andrewballantyne . Fixed